backend/app/usecase/shortlink/creator_test.go

```diff
@@ -21,11 +21,23 @@ func TestShortLinkCreatorPersist_CreateShortLink(t *testing.T) {
 	now := time.Now()
 	utc := now.UTC()
 
+	// ShortLink creator and updater methods require string pointers
+	// It is not possible to reference a literal string directly in Go
+	// To make the test suite as clean as possible, they are instead declared below and
+	// the variables are passed into the test case by reference, which is a legal semantic in Go.
+	// https://golang.org/ref/spec#Address_operators
+
+	// Set of aliases representing different classes of aliases to be tested.
 	alias := "220uFicCJj"
 	longAlias := "an-alias-cannot-be-used-to-specify-default-arguments"
 	invalidFragmentAlias := "cant-have#chr"
 	emptyAlias := ""
 
+	// Set of long links representing different classes of long links to be tested.
+	longLink := "https://www.google.com"
+	invalidLongLink := "aaaaaaaaaaaaaaaaaaa"
+	maliciousLongLink := "http://malware.wicar.org/data/ms14_064_ole_not_xp.html"
```

Comment:
Nit: We can create helper to convert string literal to pointer in the future to remove this tech debt.